tests/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+function(add_oscap_test test_script)
+	add_test(NAME ${CMAKE_CURRENT_SOURCE_DIR}/${test_script}
```

Comment:
the ctest output looks like:
```
1/74 Test  #3: /home/mpreisle/d/openscap/tests/API/CPE/lang/test_api_cpe_lang.sh ...................................   Passed    0.16 sec
```

could you remove the `/home/mpreisle/d/openscap/tests/` prefix from the test name? it would make the output slightly easier to read because the most important part is at the end of the test name. This is how it could look like after the prefix is removed:

```
1/74 Test  #3: API/CPE/lang/test_api_cpe_lang.sh ...................................   Passed    0.16 sec
```

Reply:
@mpreisler I think the test suite port to cmake is a great starting point for @dahaic 's revamp.

Regarding the prefix, that's a great idea, thank you. I'll remove the prefix.